src/ORM/Query.php

```diff
@@ -18,6 +18,7 @@
 use Cake\Database\ExpressionInterface;
 use Cake\Database\Query as DatabaseQuery;
 use Cake\Database\TypeMap;
+use Cake\Database\TypedResultInterface;
```

Comment:
@Lorenzo it seems that you just need to move this line up and ise alphabetical order to please phpcs.